meta/views.py

```diff
@@ -200,7 +200,7 @@ def get_meta_twitter_creator(self, context=None):
         return self.twitter_creator
 
     def get_meta_twitter_card(self, context=None):
-        return self.twitter_card
+        return self.twitter_card or settings.TWITTER_CARD
```

Comment:
@arthur-wsw sorry for the delay
`twitter_type` and `twitter_card` actually mean the same thing as both are rendered as `twitter:card` in template
I'd use `settings.TWITTER_TYPE` instead of adding a new setting here